api/control.proto

```diff
@@ -317,10 +317,15 @@ message JoinTokenRotation {
 	// RotateWorkerToken tells UpdateCluster to rotate the worker secret.
 	bool rotate_worker_token = 1;
 
-	// RotateManagerSecret tells UpdateCluster to rotate the manager secret.
+	// RotateManagerToken tells UpdateCluster to rotate the manager secret.
 	bool rotate_manager_token = 2;
 }
 
+message UnlockKeyRotation {
```

Comment:
Can we create a model to rotate a key by some identifier? Like `RotateKey("manager_token")`.